src/sentry/apidocs/examples/tags_examples.py

```diff
@@ -0,0 +1,37 @@
+from drf_spectacular.utils import OpenApiExample
+
+SIMPLE_TAG_DETAILS = (
```

Comment:
the api-docs error is because u have the parenthesis here (+ the above comment with the response dataclass), you can unwrap this 1 level and should be unblocked.